tests/freqai/test_freqai_interface.py

```diff
@@ -16,6 +17,9 @@
 from tests.conftest import create_mock_trades, get_patched_exchange, log_has_re
 from tests.freqai.conftest import get_patched_freqai_strategy, make_rl_config
 
+if sys.platform.startswith("win"):
```

Comment:
We will not be skipping the whole freqAI tests on windows.
that's both unnecessary, as well as danerous.

best fix your setup instead.